### PR TITLE
feat: add testing for php

### LINKED ISSUE #422: Add PHP Golden test

### PR DESCRIPTION
**Pull request by bshaffer**
Add golden tests for PHP!

Fixes https://github.com/googleapis/doc-templates/issues/422

### CODE REVIEW

File: testdata/goldens/php/xrefmap.yml [hunk 1]
```diff
@@ -0,0 +1,3106 @@
+### YamlMime:XRefMap
```
Comment (dandhlee) [inline testdata/goldens/php/xrefmap.yml:1]: This file can be removed for doc-templates goldens.

File: testdata/goldens/php/_toc.yaml [hunk 2]
```diff
@@ -0,0 +1,246 @@
+
```
Comment (dandhlee) [inline testdata/goldens/php/_toc.yaml:1]: This should be named `toc.yaml`

**Review by dandhlee — CHANGES_REQUESTED** [review]
You can update goldens with `pytest --update-goldens tests` or by running `UPDATE_GOLDENS=1 TRAMPOLINE_BUILD_FILE=./ci/run_tests.sh TRAMPOLINE_IMAGE=gcr.io/cloud-devrel-kokoro-resources/docfx ci/trampoline_v2.sh` on docker. Manual updates can be hard to catch mistakes 😓

**Comment by dandhlee** [comment]
Seems like there's still some discrepancy - could you take a look @bshaffer?

File: testdata/goldens/php/xrefmap.yml (hunk 1 above)
Comment (bshaffer) [reply testdata/goldens/php/xrefmap.yml]: done!

**Review by dandhlee — APPROVED** [review]
Thank you!

**Pull request merged**